modules/features/cu_wysiwyg/cu_wysiwyg.module

```diff
@@ -22,25 +22,27 @@ function cu_wysiwyg_wysiwyg_editor_settings_alter(&$settings, $context) {
  *   A list of pathologic paths for the current site.
  */
 function _create_cu_pathologic_paths() {
-  $cu_sid = variable_get('cu_sid', FALSE);
-  $cu_path = variable_get('cu_path', FALSE);
-  if ($cu_sid || $cu_path) {
-    $paths = array();
-    // Build hash paths
-    if ($cu_sid) {
-      $paths[] = "/$cu_sid/";
-      $paths[] = "http://www.colorado.edu/$cu_sid/";
-      $paths[] = "https://www.colorado.edu/$cu_sid/";
-    }
-    // Build real paths
-    if ($cu_path) {
-      $paths[] = "/$cu_path/";
-      $paths[] = "http://www.colorado.edu/$cu_path/";
-      $paths[] = "https://www.colorado.edu/$cu_path/";
-    }
-    // Build pathologic settings on one path per line
-    return $all_paths = join("\n", $paths);
-  }
+  // Use the global base_path and base_url set in bootstrap.inc/drupal settings initialize.
+  $base_url = $GLOBALS['base_url'];
+  $base_path = $GLOBALS['base_path'];
+
+  $paths = array();
+  
+  $paths[] = "$base_url/";
+  $paths[] = "$base_path";
+  $paths[] = "http://www.colorado.edu$base_path";
+  $paths[] = "https://www.colorado.edu$base_path";
+  
+  // Build pathologic settings on one path per line
+  $all_paths = join("\n", $paths);
+  return $all_paths;
+}
+
+/**
+ * Implements Pathologic's hook_pathologic_alter().
+ */
+function hook_pathologic_alter(&$url_params, $parts, $settings) {
+  $aPlaceToStop = 'here';
```

Comment:
What is this function doing?

Reply:
@jwfuller Sorry! I meant to take that out. I'll do that.